tools/harness-automation/autothreadharness/settings_sample.py

```diff
@@ -73,9 +77,14 @@
 
 GOLDEN_DEVICES = []
 """[(str, str)]: devices list.
-
 It should be something like [('COM1', 'OpenThread'), ('COM2', 'ARM')] for devices connected to Windows.
+For OpenThread golden devices, ser2net is also supported, just use IP:PORT for the name. For example,
```

Comment:
is this still supported?

Reply:
yes, it still supports IP:Port connection.

Reply:
are there any document introduce the use case for  the IP:Port and how to use it?